samples/gen-go/server/router.go

```diff
@@ -50,18 +52,35 @@ func (s *Server) Serve() error {
 		s.l.Info("please provide a kvconfig.yml file to enable app log routing")
 	}
 
-	if lightstepToken := os.Getenv("LIGHTSTEP_ACCESS_TOKEN"); lightstepToken != "" {
-		tags := make(map[string]interface{})
-		tags[lightstep.ComponentNameKey] = "swagger-test"
-		lightstepTracer := lightstep.NewTracer(lightstep.Options{
-			AccessToken: lightstepToken,
-			Tags:        tags,
-			UseGRPC:     true,
-		})
-		defer lightstep.FlushLightStepTracer(lightstepTracer)
-		opentracing.InitGlobalTracer(lightstepTracer)
+	if signalfxToken := os.Getenv("SIGNALFX_ACCESS_TOKEN"); signalfxToken != "" {
+		ingestUrl := os.Getenv("SIGNALFX_INGEST_URL")
+		if ingestUrl == "" {
+			ingestUrl = "https://ingest.signalfx.com"
+		}
+
+		// Create a Jaeger HTTP Thrift transport
+		transport := transport.NewHTTPTransport(ingestUrl+"/v1/trace",
+			transport.HTTPBasicAuth("auth", signalfxToken))
+
+		cfg, err := jaegercfg.FromEnv()
```

Comment:
is there a more explicit way to do this? It's not immediately clear to me what jaeger needs from the environment, and in the past we've been bitten by implicit reading of environment variables. So I'd prefer if this config were constructed directly from environment variables referenced in this file.